datumaro/plugins/data_formats/coco/base.py

```diff
@@ -115,30 +116,43 @@ def __iter__(self):
         yield from self._items.values()
 
     def _load_categories(self, json_data, *, keep_original_ids):
+        self._categories = {}
+
         if has_meta_file(self._rootpath):
             labels = parse_meta_file(self._rootpath).keys()
             self._categories = {AnnotationType.label: LabelCategories.from_iterable(labels)}
             # 0 is reserved for no class
             self._label_map = {i + 1: i for i in range(len(labels))}
-            return
-
-        self._categories = {}
+        else:
+            if self._task in [
+                CocoTask.instances,
+                CocoTask.labels,
+                CocoTask.person_keypoints,
+                CocoTask.stuff,
+                CocoTask.panoptic,
+            ]:
+                self._load_label_categories(
+                    self._parse_field(json_data, "categories", list),
+                    keep_original_ids=keep_original_ids,
+                )
 
-        if self._task in [
-            CocoTask.instances,
-            CocoTask.labels,
-            CocoTask.person_keypoints,
-            CocoTask.stuff,
-            CocoTask.panoptic,
-        ]:
-            self._load_label_categories(
-                self._parse_field(json_data, "categories", list),
-                keep_original_ids=keep_original_ids,
+            if self._task == CocoTask.person_keypoints:
+                self._load_person_kp_categories(self._parse_field(json_data, "categories", list))
+
+        # informs users if 0 is found as category id sicne 0 is reserved for no class
+        found = [
+            self._categories[AnnotationType.label][label_id].name
+            for cat_id, label_id in self._label_map.items()
+            if cat_id == 0 and self._categories[AnnotationType.label][label_id].name != "background"
+        ]
+        if found:
+            category_name = found[0]
+            log.warning(
+                "Category id of '0' is reserved for no class (background) but "
+                f"category named '{category_name}' with id of '0' is found in {self._path}. "
+                "Please be warned that annotations with category id of '0' would have `None` as label."
```

Comment:
Can we give a link to our docs here to get more details?
https://github.com/openvinotoolkit/datumaro/pull/428/files

Reply:
Added in https://github.com/openvinotoolkit/datumaro/pull/974/commits/f622aa33466623d243c6c60fafe79858e7ebe9d1